staging/src/k8s.io/cri-api/OWNERS

```diff
@@ -0,0 +1,15 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
```

Comment:
On one hand, I think we do need owners for this repository.
One the other hand, I think we may want to keep today's owner structure for CRI api itself. Today, the closest OWNERS file in the CRI directory path is [`pkg/kubelet/OWNERS`](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/OWNERS).

How about we do this?
1) Keep this top level OWNERS.
2) Add an owner file `k8s.io/cri-api/pkg/OWNERS`, which has the same content with [`pkg/kubelet/OWNERS`](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/OWNERS) with `no_parent_owners: true`?

Reply:
Done!